datasets/wit/README.md

```diff
@@ -0,0 +1,236 @@
+- es
+- et
+- eu
+- fa
+- fi
+- fr
+- fy
+- ga
+- gl
+- hr
+- hu
+- hy
+- id
+- it
+- iw
+- ja
+- ka
+- ko
+- la
+- lt
+- lv
+- mk
+- ml
+- ms
+- nl
+- nn
+- no
+- pl
+- pt
+- ro
+- ru
+- sk
+- sl
+- sr
+- sv
+- th
+- tr
+- uk
+- ur
+- vi
+- vo
+- zh
+licenses:
+- cc-by-sa-3.0
+multilinguality:
+- multilingual
+paperswithcode_id: wit
+pretty_name: Wikipedia-based Image Text
+size_categories:
+- 10M<n<100M
+source_datasets:
+- original
+- extended|wikipedia
+task_categories:
+- text-retrieval
+- image-to-text
+task_ids:
+- text-retrieval-other-text-image-retrieval
+- image-captioning
+---
+
+# Dataset Card for [Dataset Name]
+
+## Table of Contents
+- [Table of Contents](#table-of-contents)
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-fields)
+  - [Data Splits](#data-splits)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** https://github.com/google-research-datasets/wit
+- **Repository:** https://github.com/google-research-datasets/wit
+- **Paper:** https://arxiv.org/abs/2103.01913
+- **Leaderboard:** https://www.kaggle.com/c/wikipedia-image-caption
+- **Point of Contact:** krishnaps@google.com
+
+### Dataset Summary
+
+Wikipedia-based Image Text (WIT) Dataset is a large multimodal multilingual dataset. WIT is composed of a curated set of 37.6 million entity rich image-text examples with 11.5 million unique images across 108 Wikipedia languages. Its size enables WIT to be used as a pretraining dataset for multimodal machine learning models.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The dataset contains examples from all Wikipedia languages, with the following stats:
+
+Image-Text   | # Lang | Uniq. Images  | # Lang
+------------ | ------ | ------------- | ------
+total > 1M   | 9      | images > 1M   | 6
+total > 500K | 10     | images > 500K | 12
+total > 100K | 36     | images > 100K | 35
+total > 50K  | 15     | images > 50K  | 17
+total > 14K  | 38     | images > 13K  | 38
+
+## Dataset Structure
+
+### Data Instances
+
+```
+
+```
+### Data Fields
```

Comment:
Maybe we can add this [image](https://production-media.paperswithcode.com/datasets/Screenshot_2021-03-04_at_14.26.02.png) from Papers With Code to this section and add some basic descriptions to the fields (if it's too complex to explain, then feel free to reference the figure). You can check Metashift's [card](https://github.com/huggingface/datasets/tree/master/datasets/metashift) for examples of adding figures to Markdown.

We can keep the reference to the table with values from the paper.